target/linux/mediatek/mt7629/base-files/lib/preinit/02_sysinfo_fixup

```diff
@@ -0,0 +1,18 @@
+. /lib/functions.sh
+
+do_sysinfo_mediatek_fixup() {
+	local model=""
+
+	case $(board_name) in
+	netgear,ex6250-v2)
+		local part=$(find_mtd_part Config)
+		local config_model=$(strings "$part" | sed -n 's/^bmodel=//p')
+		# EX6250v2, EX6400v3, EX6470, EX6410v2
+		[ -n "$config_model" ] && model="Netgear $config_model"
```

Comment:
I prefer if we do not manually update this even when it does not show the correct name. If the hardware is anyway the same it should not make a big difference.

Reply:
Done. I also deleted the ath79 version of this file for consistency.